docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -1122,6 +1122,50 @@ public class ProjectPermissionChecker {
 TIP: Permission checks run by default on event loops.
 Annotate a permission checker method with the `io.smallrye.common.annotation.Blocking` annotation if you want to run the check on a worker thread.
 
+Matching between the `@PermissionsAllowed` values and the `@PermissionChecker` value is based on a string equality.
```

Comment:
```suggestion
Matching between the `@PermissionsAllowed` values and the `@PermissionChecker` value is based on string equality.
```

Reply:
Though I honestly not 100% sure :-), I just have a feeling we can drop `a` here :-), Michal feel free to ignore this suggestion if you'd like

Reply:
it feels bit better without `a`, thanks